python/paddle/quantization/__init__.py

```diff
@@ -12,35 +12,47 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-from ...fluid.contrib.slim.quantization.imperative.ptq_config import (
+from ..fluid.contrib.slim.quantization.imperative.ptq_config import (
     PTQConfig,
     default_ptq_config,
 )
-from ...fluid.contrib.slim.quantization.imperative.ptq_quantizer import (
+from ..fluid.contrib.slim.quantization.imperative.ptq_quantizer import (
     BaseQuantizer,
 )
-from ...fluid.contrib.slim.quantization.imperative.ptq_quantizer import (
+from ..fluid.contrib.slim.quantization.imperative.ptq_quantizer import (
     AbsmaxQuantizer,
 )
-from ...fluid.contrib.slim.quantization.imperative.ptq_quantizer import (
+from ..fluid.contrib.slim.quantization.imperative.ptq_quantizer import (
     PerChannelAbsmaxQuantizer,
 )
-from ...fluid.contrib.slim.quantization.imperative.ptq_quantizer import (
+from ..fluid.contrib.slim.quantization.imperative.ptq_quantizer import (
     KLQuantizer,
 )
-from ...fluid.contrib.slim.quantization.imperative.ptq_quantizer import (
+from ..fluid.contrib.slim.quantization.imperative.ptq_quantizer import (
     HistQuantizer,
 )
-from ...fluid.contrib.slim.quantization.imperative.ptq_quantizer import (
+from ..fluid.contrib.slim.quantization.imperative.ptq_quantizer import (
     SUPPORT_ACT_QUANTIZERS,
 )
-from ...fluid.contrib.slim.quantization.imperative.ptq_quantizer import (
+from ..fluid.contrib.slim.quantization.imperative.ptq_quantizer import (
     SUPPORT_WT_QUANTIZERS,
 )
-from ...fluid.contrib.slim.quantization.imperative.ptq_registry import (
+from ..fluid.contrib.slim.quantization.imperative.ptq_registry import (
     PTQRegistry,
 )
-from ...fluid.contrib.slim.quantization.imperative.ptq import ImperativePTQ
-from ...fluid.contrib.slim.quantization.imperative.qat import (
+from ..fluid.contrib.slim.quantization.imperative.ptq import ImperativePTQ
+from ..fluid.contrib.slim.quantization.imperative.qat import (
     ImperativeQuantAware,
 )
+
+
+from .config import QuantConfig
+from .base_quanter import BaseQuanter
+from .factory import quanter
+from .quanters import FakeQuanterWithAbsMaxObserver
+
+__all__ = []
+__all__ += ["QuantConfig"]
+__all__ += ["BaseQuanter"]
+__all__ += ["quanter"]
+__all__ += ["FakeQuanterWithAbsMaxObserver"]
```

Comment:
it's better to use one python list to represent `__all__` instead of merging many lists with `+=`, more intuitive and better monitoring.

Reply:
`QuantConfig`, `BaseQuanter`, `quanter` and `FakeQuanterWithAbsMaxObserver` also in other file's `__all__`, this is prohibited, which will cause the same API to have two recommended paths for users with inconsistent usage and easy confusion.

Reply:
Thx. Done.